math-magicians/README.md

```diff
@@ -0,0 +1,70 @@
+# Getting Started with Create React App
```

Comment:
- [ ] Please, kindly implement the [Microverse Readme Template](https://github.com/microverseinc/readme-template) on your readme file, instead of the CRA "getting started" readme.  This Microverse readme template is required for all projects.  Please, include all the sections except for the "tests" and "live demo" if not required by the project.

That way, your app will be well documented, and we will meet this particular requirement 💪 

- [ ] OPTIONAL:  I highly recommend please don't upload the repo inside a folder.  In order to correct this, you can move everything to the parent folder, then delete the empty folder  Then, npm install again to update the dependencies, commit and push.  That way, the repo will look normal (not a single folder).